tests/test_gail.py

```diff
@@ -38,15 +38,60 @@ def test_gail(expert_env):
             obs = env.reset()
     del dataset, model
 
-
-def test_generate_pendulum():
+#pendulum single env support lerner single env
+def test_generate_t0():
     model = SAC('MlpPolicy', 'Pendulum-v0', verbose=0)
-    generate_expert_traj(model, 'expert_pendulum', n_timesteps=1000, n_episodes=10)
-
+    generate_expert_traj(model, 'expert_pendulum', n_timesteps=1000, n_episodes=10,
+                         image_folder='test_recorded_images')
 
-def test_generate_cartpole():
+#cartpole single env support lerner single env
+def test_generate_t1():
```

Comment:
Please use `pytest.parametrized` to avoid duplicated code (see [here](https://github.com/hill-a/stable-baselines/blob/master/tests/test_gail.py#L15) for an example.

Reply:
Done
This is such a better solution.